cooperator_website/controllers/main.py

```diff
@@ -439,9 +439,8 @@ def share_subscription(self, **kwargs):  # noqa: C901 (method too complex)
                 values["company_register_number"] = re.sub(
                     "[^0-9a-zA-Z]+", "", kwargs.get("company_register_number")
                 )
-            subscription_id = sub_req_obj.sudo().create_comp_sub_req(values)
-        else:
-            subscription_id = sub_req_obj.sudo().create(values)
+
+        subscription_id = sub_req_obj.sudo().create(values)
```

Comment:
Can you either remove `def create_comp_sub_req` or log a warning there to make the depreciation stand out ?

Reply:
created #391. this is for version 12.0. if you approve this pr, instead of merging it, i’ll add the changes to #375 (12.0) and to #376 (14.0), and will merge those.